SwiftScripts/Sources/Codegen/Target.swift

```diff
@@ -6,6 +6,7 @@ enum Target {
   case gitHub
   case upload
   case animalKingdom
+  case ccnAnimalKingdom
```

Comment:
This file and `Module.swift` were moved in my latest PR. But I don't think we need a separate `ccnAnimalKingdom` target. I don't really see the value in that. If CCN works and is supported and everything is going into the same generated module, then lets just do it all the time.

In that case, all you need is the `target.ccnEnabled` and we are good to go!

Reply:
I've deleted `ccnAnimalKingdom` and made CCN enabled all the time for the `animalKingdom` target, but left it disabled for the other targets. I think that's what you wanted, but let me know if I should do something else!